cwms-data-api/src/main/java/cwms/cda/api/watersupply/WaterPumpDisassociateController.java

```diff
@@ -86,41 +87,18 @@ public WaterPumpDisassociateController(MetricRegistry metrics) {
     public void handle(@NotNull Context ctx) {
         try (Timer.Context ignored = markAndTime(DELETE)) {
             DSLContext dsl = getDslContext(ctx);
-            boolean deleteAccounting = Boolean.parseBoolean(ctx.queryParam(METHOD));
+            boolean deleteAccounting = Boolean.parseBoolean(requiredParam(ctx, METHOD));
```

Comment:
use `requiredParamAs` and pass in the Boolean.class

Reply:
Replaced with requiredParamAs